fedimint-server-core/Cargo.toml

```diff
@@ -1,15 +1,15 @@
 [package]
-name = "fedimint-server-core"
-version = { workspace = true }
 authors = { workspace = true }
 description = { workspace = true }
 documentation = { workspace = true }
 edition = { workspace = true }
-readme = { workspace = true }
 homepage = { workspace = true }
-repository = { workspace = true }
-license = { workspace = true }
 keywords = { workspace = true }
+license = { workspace = true }
+name = "fedimint-server-core"
+readme = { workspace = true }
+repository = { workspace = true }
+version = { workspace = true }
```

Comment:
How did that reordering happen? Bug in CI that allowed a PR to be merged with incorrectly sorted `Cargo.toml`?

Reply:
`cargo-sort` only cares about order of `[dependencies]`. And even there it seems it's not exactly strict about common-prefix dependencies.